CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+v1.0.0-pre3
+-------
+
+* **Release date**: 2016-09-16 (dev build only as public preview)
+* **Requires:** [tileserver v0.8.0-pre2](https://github.com/mapzen/tileserver/releases/tag/v0.8.0-pre2) and [tilequeue v0.11.0-pre2](https://github.com/mapzen/tilequeue/releases/tag/v0.11.0-pre2) and [mapbox-vector-tile v0.5.0](https://pypi.python.org/pypi/mapbox-vector-tile/0.5.0).
+
+* Removed "not equals" YAML rule, which can be expressed using the other "equals" and "not" operators. [PR #1044](https://github.com/tilezen/vector-datasource/pull/1044).
+* **BREAKING** Rename `sort_key` to `sort_rank`. [PR #1049](https://github.com/tilezen/vector-datasource/pull/1049).
+* **BREAKING** Add `/mapzen` prefix to tilejson tiles URL. [PR #1047](https://github.com/tilezen/vector-datasource/pull/1047).
```

Comment:
This has been relaxed to being optional in dev. That being said it's probably best to leave it in here as breaking because we plan on changing it to be a requirement eventually.